tests/bearlib/languages/documentation/documentation_extraction_testdata/default.c

```diff
@@ -33,7 +33,8 @@ int foobar(int x) {
 }
 
      /** line 1
-      *  line2 */
+      *  line2
+      */
```

Comment:
we can investigate this later. For further info.
https://github.com/coala/coala/issues/4548